src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SafeEvpPKeyHandle.OpenSsl.cs

```diff
@@ -177,5 +193,84 @@ public static SafeEvpPKeyHandle OpenPublicKeyFromEngine(string engineName, strin
 
             return Interop.Crypto.LoadPublicKeyFromEngine(engineName, keyId);
         }
+
+        /// <summary>
+        ///   Open a named public key using a named OpenSSL <code>provider</code>.
+        /// </summary>
+        /// <param name="providerName">
+        ///   The name of the <code>provider</code> to process the key open request.
+        /// </param>
+        /// <param name="keyUri">
+        ///   The URI of the key to open.
+        /// </param>
+        /// <returns>
+        ///   The opened key.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="providerName"/> or <paramref name="keyUri"/> is <see langword="null" />.
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <paramref name="providerName"/> or <paramref name="keyUri"/> is the empty string.
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   the key could not be opened via the specified provider.
+        /// </exception>
+        /// <remarks>
+        ///   <para>
+        ///     Both provider name and key URI must be trusted inputs.
```

Comment:
This is probably a quirk of English, but as abstract concepts they need a determiner... but if they're proper nouns they don't:

with determiners
```suggestion
        ///     Both the provider name and the key URI must be trusted inputs.
```

as proper nouns

```suggestion
        ///     Both <paramref name="providerName" /> and <paramref name="keyUri" /> must be trusted inputs.
```